src/Security/Member.php

```diff
@@ -447,59 +422,41 @@ public function isPasswordExpired()
     }
 
     /**
-     * Logs this member in
+     * @deprecated Use Security::setCurrentUser() or IdentityStore::logIn()
      *
-     * @param bool $remember If set to TRUE, the member will be logged in automatically the next time.
      */
-    public function logIn($remember = false)
+    public function logIn()
     {
-        $this->extend('beforeMemberLoggedIn');
-
-        self::session_regenerate_id();
+        Deprecation::notice('5.0.0',
```

Comment:
Please use the proper deprecation API, and ensure that your deprecation notice provides the user with the correct upgrade path.

Deprecation::notice("5.0.0", "This method is deprecated and now does not persist. Please use <some new api>");